### ISSUE TITLE
Color contrast ratio for the focus border of the cancel button and its background is 2.589:1 which is less than 3:1 in the create new database project dialog: A11y_SQL Azure Data Studio_Create new database project_AI4D

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#26059]
Please do not close this bug. This bug should only be closed by Trusted Tester after veriﬁcation 
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug ﬁxing faster and easier." 

### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Nov2024; #SQL Azure Data Studio; #WCAG1.4.3;  #Win32; #Benchmark; #FTP; #AI4D;#ColorContrast;

### Environment Details:
Application Name: SǪL Azure Data Studio 
Edition: Windows 11 Enterprise 
Version: 24H2 
Installed: 22-10-2024 
OS build: 26100.2033 
Experience: Windows Feature Experience Pack 1000.26100.23.0 

### Steps to Reproduce
1.Launch the application-Azure Database Project and install extension SǪL Database Projects. 
2.Tab to “Connections” tab item present in the left menu and Hit Enter Key  
3.Tab to "Create New Project" button or "+" button and Hit ENTER key. 
4."Create new database project” dialog will get open. 
5. Tab to “Cancel” button in the dialog. 
6. Open accessibility insights for windows and check for the color contrast ratio of the focus border of the cancel button and its background  
7. Similar issue observed throughout the application

### Actual Results:
Color contrast ratio for the focus border of the cancel button and its background is 2.589:1 which is less than 3:1 in the create new database project dialog.

### Expected Results:
Color contrast ratio for the focus border of the cancel button and its background should be greater than or equal to 3:1 

### 
### User Impact:
Low vision users will not be able to see the focus border for the controls if the color contrast ratio is less than 3:1. 

### Attachment:
![Image](https://github.com/user-attachments/assets/85b9937d-43fa-4d79-9879-4dd21de1f23c)
![Image](https://github.com/user-attachments/assets/18515942-1892-419c-b6c8-0940b7e192bf)
![Image](https://github.com/user-attachments/assets/704213c4-ed0d-4b74-a9de-f1d72898be6b)
![Image](https://github.com/user-attachments/assets/58514eac-2356-4446-9543-cbe8a8e14a21)
